### PR TITLE
Add detection example from image

### PR DESCRIPTION
**Pull request by fmassa**
Addresses https://github.com/szagoruyko/cifar.torch/issues/11

### CODE REVIEW

File: example_classify.lua [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+if #view > 0 then
+  view[1].numInputDims = 3
+end
+
+local cls = {'airplane', 'automobile', 'bird', 'cat',
+             'deer', 'dog', 'frog', 'horse', 'ship', 'truck'}
+
+for i = 2, #arg do
+  local img = image.load(arg[i], 3, 'float'):mul(255)
+  local name = arg[i]:match( "([^/]+)$" )
```
Comment (szagoruyko) [inline example_classify.lua]: what's this for?

Comment (fmassa) [inline example_classify.lua]: random playing with the names ;)
will take it off

**Comment by fmassa** [comment]
Removed the unnecessary code

**Comment by szagoruyko** [comment]
thanks Francisco, one more thing, can you do

``` lua
local model_path = arg[1]
local image_paths = tablex.sub(arg, 2, -1)
```

for readability

somewhere here https://github.com/szagoruyko/cifar.torch/pull/13/files#diff-404e18017e34b820b286d38260f987f0R14

**Comment by fmassa** [comment]
Sure, no problem.
Just pushed the changes

**Comment by fmassa** [comment]
Wait, forgot to import `pl`

**Comment by fmassa** [comment]
Done

**Comment by szagoruyko** [comment]
thanks!

**Pull request merged**